### PR TITLE
adding timestamp features

### PR DESCRIPTION
**Pull request by clewis7**
addresses #127 

human readable timestamp added when item added to dataframe and ran time stamp is added when item is ran

just trying to get back into the swing of things, thought this would be a good one to implement!!

### CODE REVIEW

**Review by kushalkolar — CHANGES_REQUESTED** [review]

**Comment by kushalkolar** [comment]
linux test failing but windows test passing? I don't even :open_mouth:

File: mesmerize_core/algorithms/cnmf.py [hunk 1]
```diff
@@ -139,7 +141,8 @@ def run_algo(batch_path, uuid, data_path: str = None):
     # Add dictionary to output column of series
     df.loc[df["uuid"] == uuid, "outputs"] = [d]
     # Add ran timestamp to ran_time column of series
-    df.loc[df["uuid"] == uuid, "ran_time"] = time.asctime(time.localtime(time.time()))
+    df.loc[df["uuid"] == uuid, "ran_time"] = datetime.now().isoformat(timespec="milliseconds")
```
Comment (kushalkolar) [inline mesmerize_core/algorithms/cnmf.py:144]: does this return in the local timezone?

Comment (kushalkolar) [reply mesmerize_core/algorithms/cnmf.py:144]: how much extra info is kept with this, we really only want YYYY-MM-DD-HH-MM-SS

Comment (clewis7) [reply mesmerize_core/algorithms/cnmf.py:144]: will fix to only return up to seconds

returns local time zone as established on the computer

Comment (clewis7) [reply mesmerize_core/algorithms/cnmf.py:144]: file:///home/caitlin/Desktop/timestamp_ex.png

Comment (clewis7) [reply mesmerize_core/algorithms/cnmf.py:144]: timestamp_ex

Comment (kushalkolar) [reply mesmerize_core/algorithms/cnmf.py]: is `/` the default separator for this? Is there a way to get a different separator? maybe `T`?

**Comment by clewis7** [comment]
I think everything should be fixed here, let me know if there is anything else :)

**Review by kushalkolar — APPROVED** [review]
LGTM, will merge if CI passes

**Comment by kushalkolar** [comment]
CI is failing because new numpy deprecated `numpy.int` and caiman uses them :man_facepalming:

**Comment by kushalkolar** [comment]
thanks! @clewis7

**Pull request merged**